### PR TITLE
Update README.md

### LINKED ISSUE #4: Create README.md
Adds a readme Following a similar format:  https://gist.github.com/PurpleBooth/109311bb0361f32d87a2

### PR DESCRIPTION
**Pull request by christopher-alba**
Readme should follow a similar layout to this one: https://gist.github.com/PurpleBooth/109311bb0361f32d87a2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,51 @@
...
+  --> create a new branch
+  
+  --> proceed to add your contributions
+  
+  --> create a pull request on the SE701-T5/Frontend repository from your forked repo. 
+  
+For more details, please read our CONTRIBUTING.md file.
+
+
+## Authors:
```
Comment (Tvo465) [inline README.md]: Have the names in alphabetical order?
But besides that everything else seems good.

**Review by Tvo465 — CHANGES_REQUESTED** [review]

**Review by Tvo465 — DISMISSED** [review]
Looks good :)

**Comment by yuchendong13** [comment]
Looks good to me!

**Review by yuchendong13 — DISMISSED** [review]

**Review by R055A — CHANGES_REQUESTED** [review]
* Why run ```npm install``` before forking and cloning the repository?
* It could be better to change the heading "Afterwards:" instead to "### Setup" or "### Install"
* It may be simpler 
    * to use:
        * run ```npm install``` 
   *  instead of (we don't know if an IDE is being used):
        * "in your IDE's terminal run "npm install" to install all the dependencies"
* you could reference to the CONTRIBUTING.md file with a link in ### Contributions, such as:
    * refer to [CONTRIBUTING.md](../../../.github/CONTRIBUTING.md) - the link won't work in this comment
* "## Contributions" could be renamed "## Contribute"
* "### Deployment" could come after "### Starting up react app"
* "### Starting up react app" could be renamed "### Run"

File: README.md [hunk 2]
```diff
@@ -1,2 +1,51 @@
...
+https://docs.microsoft.com/en-us/windows/dev-environment/javascript/nodejs-on-windows
+
+#### MacOS
+https://www.newline.co/@Adele/how-to-install-nodejs-and-npm-on-macos--22782681
+
+#### Linux
+https://linuxize.com/post/how-to-install-node-js-on-ubuntu-18.04/
+
+### Contribute
+To get started to contributing, please create a new issue in the "Issues" tab describing the new bug or feature you want to add, and add it to the "Projects" Kanban board.
```
Comment (r055a) [inline README.md]:
```suggestion
Refer to [CONTRIBUTING.md](../../contribution-guidelines/.github/CONTRIBUTING.md).
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,51 @@
...
+#### MacOS
+https://www.newline.co/@Adele/how-to-install-nodejs-and-npm-on-macos--22782681
+
+#### Linux
+https://linuxize.com/post/how-to-install-node-js-on-ubuntu-18.04/
+
+### Contribute
+To get started to contributing, please create a new issue in the "Issues" tab describing the new bug or feature you want to add, and add it to the "Projects" Kanban board. 
+
+Afterwards:
```
Comment (r055a) [inline README.md]:
```suggestion
### Install
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,51 @@
...
+  
+  --> run ```npm install```
+  
+  --> create a new branch
+  
+  --> proceed to add your contributions
+  
+  --> create a pull request on the SE701-T5/Frontend repository from your forked repo. 
+  
+For more details, please read our CONTRIBUTING.md file.
```
Comment (r055a) [inline README.md]:
```suggestion
 Inside the project, to install dependencies
 ```npm install```
```

It seems redundant to repeat what is already instructed in CONTRIBUTING.md

File: README.md [hunk 5]
```diff
@@ -1,2 +1,51 @@
...
+
+### Run
+To start viewing the frontend run ```npm start```. This should start up the frontend and open a browser automatically. View the site on http://localhost:3000/.
```
Comment (r055a) [inline README.md]:
```suggestion

Inside the project, to run in development mode
```npm start``` 

View browser in development mode
```http://localhost:3000/```
```

File: README.md [hunk 6]
```diff
@@ -1,2 +1,51 @@
...
+
+### Deployment
+Frontend repo's main branch will be automatically deployed to Netlify.
+
+## Frontend Contributors:
```
Comment (r055a) [inline README.md:43]: I suggest adding contributors individually after they have made contributions

**Review by R055A — CHANGES_REQUESTED** [review]

File: README.md [hunk 7]
```diff
@@ -1,2 +1,51 @@
...
+  --> run ```npm install```
+  
+  --> create a new branch
+  
+  --> proceed to add your contributions
+  
+  --> create a pull request on the SE701-T5/Frontend repository from your forked repo. 
+  
+For more details, please read our CONTRIBUTING.md file.
+
```
Comment (r055a) [inline README.md:30]:
```suggestion

### Test
Inside the project, to run automated testing
```npm test```

```

We will want to include instructions for running automated testing

**Review by R055A — CHANGES_REQUESTED** [review]

File: README.md [hunk 8]
```diff
@@ -1,2 +1,51 @@
...
+### Deployment
+Frontend repo's main branch will be automatically deployed to Netlify.
+
+## Frontend Contributors: 
+- Christopher Alba
+- Trinh Vo
+- Xiaoqi
+- Yu Chen
+
```
Comment (hpar461) [inline README.md:49]: Please add me on the list of contributors. 

```suggestion
- Hyung

```

**Review by hpar461 — DISMISSED** [review]

File: README.md (hunk 8 above)
Comment (christopher-alba) [reply README.md:49]: Yessir

**Review by Xiaoqi510 — DISMISSED** [review]

**Review by hpar461 — DISMISSED** [review]

File: README.md [hunk 9]
```diff
@@ -1,2 +1,49 @@
...
+- install git: https://git-scm.com/downloads
+- install node and npm using node version manager (NVM): 
+    * Windows: https://docs.microsoft.com/en-us/windows/dev-environment/javascript/nodejs-on-windows
+
+    * MacOS: https://www.newline.co/@Adele/how-to-install-nodejs-and-npm-on-macos--22782681
+
+    * Linux: https://linuxize.com/post/how-to-install-node-js-on-ubuntu-18.04/
+
+### Contribute
+Refer to [CONTRIBUTING.md]().
```
Comment (r055a) [inline README.md]:
```suggestion
Refer to [CONTRIBUTING.md](../../contribution-guidelines/.github/CONTRIBUTING.md).
```
The link is missing

**Review by R055A — CHANGES_REQUESTED** [review]
Great improvements!

**Review by hiin3d55 — COMMENTED** [review]
LGTM!

**Review by R055A — APPROVED** [review]

**Review by hpar461 — APPROVED** [review]

**Review by yuchendong13 — APPROVED** [review]

**Review by Xiaoqi510 — APPROVED** [review]

**Review by Tvo465 — APPROVED** [review]
Looks great

**Review by hiin3d55 — APPROVED** [review]
LGTM!

**Review by AlexMiao7 — APPROVED** [review]

**Comment by qzho236** [comment]
Very clear and helpful!

**Review by qzho236 — APPROVED** [review]

**Pull request merged**